modules/src/core/ics04_channel/packet.rs

```diff
@@ -106,7 +113,7 @@ pub struct Packet {
     pub destination_channel: ChannelId,
     #[serde(serialize_with = "crate::serializers::ser_hex_upper")]
     pub data: Vec<u8>,
-    pub timeout_height: Height,
+    pub timeout_height: TimeoutHeight,
     pub timeout_timestamp: Timestamp,
```

Comment:
Should we add a similar `TimeoutTimestamp` newtype for `Timestamp` in the future (in a different PR)?

Reply:
As discussed, `Timestamp` already serves this purpose.